app/Models/Post.php

```diff
@@ -157,7 +157,7 @@ public function toBlinkPayload()
         return [
             'id' => $this->id,
             'signup_id' => $this->signup_id,
-            'quantity' => $this->signup->quantity,
+            'quantity' => $this->quantity,
```

Comment:
will this break anything in v2 @sbsmith86 ?

Reply:
nope nope! it will just return whatever is in that column (`null` for most things until the script runs)

Reply:
ah right! Thanks for confirming!